src/locales/scripts/json-to-pot.js

```diff
@@ -0,0 +1,176 @@
+// More about the structure of .po files:
+// https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html#PO-Files
+
+/*
+ * white-space
+ #  translator-comments
+ #. extracted-comments
+ #: reference…
+ #, flag…
+ #| msgid previous-untranslated-string
+ msgid untranslated-string
+ msgstr translated-string
+ */
+const getParsedVueFiles = require('./parse-vue-files.js')
+const json = require('../en.json')
+const fs = require('fs')
+
+const curlyRegex = new RegExp('{[a-zA-Z-]*}')
+const containsCurlyWord = (string) => curlyRegex.test(string)
+const checkStringForVars = (string) =>
+  containsCurlyWord(string) ? '(Do not translate words between ###)' : ''
```

Comment:
```suggestion
  containsCurlyWord(string) ? ' (Do not translate words between ###)' : ''
```

Reply:
Adding a space here, instead of the main script, removes unnecessary spaces at the end of lines which don't contain variables with curly brackets.